srcpkgs/neovim/template

```diff
@@ -12,8 +12,8 @@ short_desc="Fork of Vim aiming to improve user experience, plugins and GUIs"
 maintainer="Steve Prybylski <sa.prybylx@gmail.com>"
 license="Apache-2.0, custom:Vim"
 homepage="https://neovim.io"
-distfiles="https://github.com/neovim/neovim/archive/v${version}.tar.gz"
-checksum=b4484e130aa962457189f3dee34b8481943c1e395d2d684c6f8b91598494d9ec
+distfiles="https://github.com/neovim/neovim/archive/refs/tags/v${version}.tar.gz"
```

Comment:
Is there a specific need to add `refs/tags/` in `distfiles`?

Reply:
Makes the link consistent with what github release page gives, and avoids potential ambiguity

Reply:
When /archive is used and a tag and branch of the same name exist, GitHub will not know which one we want. This bit us in the past and GitHub now gives this URL instead, so we are sometimes switching to it.
It's not anything important, but when someone switches to it, we let it go through and when someone tries to switch back to archive, we ask them to not do that.